### DISCUSSION TITLE
Convert Order to XML

### DISCUSSION THREAD

**Discussion by kswanner** [abuzuhri/Amazon-SP-API-CSharp#220] [General]
I'm trying to convert an Order object to XML, but it fails because **MarketplaceTaxInfo** does not have a _parameterless_ constructor. I haven't gone through all of the classes, so this is the only one I have come to thus far. Would it be possible to provide parameterless constructors for all classes? Is there another way I can get an XML representation of the Order object?

**Comment by abuzuhri** [comment]
Hi @kswanner 

Problem fixed in v[1.3.17](https://www.nuget.org/packages/CSharpAmazonSpAPI/1.3.17)

**Reply by kswanner** [reply]
Thanks for the quick response. This fixed the problem with MarketplaceTaxInfo, but now I'm getting it with BuyerInfo. Here is a list of the classes in Order that have no parameterless constructor:

- BuyerCustomizedInfoDetail
- BuyerInfo

Can you please add parameterless constructors to these classes?

**Reply by abuzuhri** [reply]
Problem fixed in v[1.3.18](https://www.nuget.org/packages/CSharpAmazonSpAPI/1.3.18)

**Reply by kswanner** [reply]
Thank you! That worked.
